packages/playwright-test/src/worker/workerMain.ts

```diff
@@ -369,8 +369,13 @@ export class WorkerMain extends ProcessRunner {
       await testInfo._runFn(async () => {
         // Now run the test itself.
         debugTest(`test function started`);
+        setRunningTestBody(true);
```

Comment:
Let's assign a property on `TestInfoImpl` instead.

Reply:
Added one on the context instead.